src/components/profile/ProfileDataTable.js

```diff
@@ -141,6 +141,9 @@ const ProfileDataTable = ({
 
 ProfileDataTable.defaultProps = {
   errors: {},
+  storedProfile: {},
```

Comment:
as we agreed some time ago, we don't use defaultProps on functional components, using JS function params with defaults instead:

```javascript
import { noop } from 'lodash'

const defaultErrors = {}
const defaultProfile = {}
const defaultStoredProfile = {}

const ProfileDataTable = ({ errors = defaultErrors, storedProfile = defaultStoredProfile, profille = defaultProfile, setLockSubmit = noop  })
```

Reply:
Done